### DISCUSSION TITLE
Logging in with deleted gmail account

### DISCUSSION THREAD

**Discussion by chosenn511** [community/community#22514] [New to GitHub]
<p>Hello, I would like to log in to my previous github account connected with deleted gmail account. Since i deleted my gmail account a while ago, reactivating account is no longer possible. I tried to recreate a gmail account with the name i used again but gmail didn’t allow this behavior either :frowning:<br>
What I want to do is logging into my previous github account WITHOUT verification process using  my deleted gmail account ( since i cannot access to it…! )<br>
Before I write this topic, I searched up topics with similar issues but they seemed to be dealt privately.</p>
<p>If someone had similar issue and solved this problem or if a staff from github could reach me, it would be a great help! Thank you.</p>

**Comment by jfullstackdev** [comment]
<p>why would you want to still use your old email? why would you delete your email if you know it’s associated with your GitHub account?</p>
<p>seems your account has no activities either,</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/c/8c609bf82051f7aa1578ba0249cfc4bc963edab6.png" data-download-href="/uploads/short-url/k1PPtqFQbIA3gDdxgSMdVIjgVN4.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181071570-09492334-e850-4276-ac10-6f7cd965a41e.png" alt="image" data-base62-sha1="k1PPtqFQbIA3gDdxgSMdVIjgVN4" width="558" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/c/8c609bf82051f7aa1578ba0249cfc4bc963edab6_2_558x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/8/c/8c609bf82051f7aa1578ba0249cfc4bc963edab6_2_837x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/8/c/8c609bf82051f7aa1578ba0249cfc4bc963edab6_2_1116x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/c/8c609bf82051f7aa1578ba0249cfc4bc963edab6_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1162×1040 108 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>why not simply create a new email and a new GitHub Account?</p>

**Comment by chosenn511** [comment]
<p>Hello, first of all, thanks for your attention.</p>
<p>What I am trying to do is to move repositories in my old github account to currently logged-in github account.<br>
So, I really need a solution to access to my old github account, without verification process using my deleted gmail account. I don’t know why I have to explain the detailed purpose though hope this answer helps your curiosity :slight_smile:</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>you can try reaching the group assigned for accounts, and hopefully they will do something</p>
<p><a href="https://support.github.com/contact/account">https://support.github.com/contact/account</a></p>

**Comment by alexblackkot9** [comment]
I'm sorry to hear that you can no longer access your deleted Gmail account in order to log into your GitHub account. That must be really frustrating. Unfortunately, I haven't faced such a problem. But I think you should [buy pva accounts](https://bestpvaaccounts.com/) from a reliable provider to take advantage of additional features. Such accounts have a two-factor identification, which will guarantee the security of your data. If you need further help, feel free to contact the GitHub staff. Good luck, and I hope you'll be able to log in soon!

**Reply by shakram02** [reply]
this is a spam URL

**Comment by Xelliese** [comment]
I had the same problem when my Gmail account got deleted, and I couldn't log into my old GitHub account. It's really annoying, especially when making a new Gmail with the same name doesn't work. I couldn't get my GitHub account back and had to make a new one.

To avoid this mess again, I started buying old Gmail accounts because they are safer and help me keep access to important accounts. This way, I don't worry about losing access again.

You might find this helpful too. Check out https://digitalstorenet.com/product/old-gmails/ for more info on old Gmail accounts. It might help you prevent these issues in the future.

**Comment by PraiseTheFool-LOTM** [comment]
could you also please look into my account which is linked to rithvikganeshkonapala@gmail.com google account which has been deleted a few years ago....i have two repositories uploaded there and want to access them but my gmail got deleted for some reason!!! please help
